CONTRIBUTING.md

```diff
@@ -0,0 +1,46 @@
+## How to contribute ##
+
+The following are the guidelines, everyone should use to contribute to utPLSQL.  
+Changes are welcome from all members of the Community. 
+
+## Getting Started ##
+
+1. Create a [GitHub Account](https://github.com/join).
+2. Fork the utPLSQLRepository and setup your local Repository.
+     * Each of of the steps below are detailed [How to Fork](https://help.github.com/articles/fork-a-repo) article!
+     * Clone your Fork to your local machine.
+     * Configure "upstream" remote to the [master utPLSQL repository](https://github.com/utPLSQL/utPLSQL.git).
+3. For each change you want to make:       
+     * Create a new branch for your change. 
+     * Make your change in your new branch. 
+         * Although changes can be made in the master branch, it easier long term if a new branch is used.
+     * **Verify code compiles and unit tests still pass.** 
+         * The quickest way to have a Pull Request not be accepted, is to submit code that does not compile or pass tests.
+     * Commit change to your local repository.
+     * Push change to your remote repository
+     * Submit a [Pull Request](https://help.github.com/articles/using-pull-requests).
+     * Note: local and remote branches can be deleted after pull request has been accepted.
+
+**Note:** Getting changes from others requires [Syncing your Local repository](https://help.github.com/articles/syncing-a-fork) with Master utPLSQL repository.    This can happen at any time.
+
+
+## Dependencies ##
+
+* Oracle Version XX.X or Later
```

Comment:
At the moment, there isn't officially a minimum version - in theory at least it should work on pretty much any version (since PL/SQL was introduced). This will change with the next version, but no decision's been made yet about what the minimum will be